stratum/hal/bin/barefoot/docker/start-stratum-container.sh

```diff
@@ -38,10 +38,12 @@ if [ -n "$CHASSIS_CONFIG" ]; then
 fi
 
 # Start Stratum.
+# --network host: the BSP needs access to the usb0 interface.
```

Comment:
@bocon13 @Yi-Tseng

I found no other way yet. For reference: the BSP will do a http request against the BMC in `bf_pltfm_bd_type_init`.

If this is our solution, we have to update the run README.